card/mod/02_basic_types/set/all/all_css.rb

```diff
@@ -1,7 +1,7 @@
 
 format :css do
 
-  def get_inclusion_defaults nested_card
+  def get_nest_defaults nested_card
```

Comment:
Unused method argument - `nested_card`. If it's necessary, use `_` or `_nested_card` as an argument name to indicate that it won't be used. You can also write as `get_nest_defaults(*)` if you want the method to accept any arguments but don't care about them.